arangodb-net-standard/AdminApi/Models/EngineAlias.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ArangoDBNetStandard.AdminApi.Models
+{
+    public class EngineAlias
+    {
+        public EngineAliasIndex Indexes { get; set; }
```

Comment:
Do you think we should make this a `Dictionary<string, string>` so that we don't have to update the library in case new aliases are added? It's a trivial thing though, I suppose it will rarely happen.

Reply:
Very good point. I've made the change.